frontend/src/hooks/providers/EthersJS.ts

```diff
@@ -0,0 +1,36 @@
+//import { Contract, ContractInterface, utils } from "ethers"
+import { Web3Provider, ExternalProvider } from '@ethersproject/providers'
+import { Provider as InterfaceProvider, GAS_PRICE_LIMIT } from './Provider'
+
+
+export class EthersJS implements InterfaceProvider<Web3Provider> {
+  library: Web3Provider
+  account: any
+
+  readonly provider: ExternalProvider
+  //contract?: Contract
+
+  constructor(provider: ExternalProvider) {
+    this.library = new Web3Provider(provider) as Web3Provider
+    this.provider = provider
+  }
+
+  getLibrary(): Web3Provider {
+    return this.library
+  }
+
+  async getBalance(): Promise<Number> {
+    return Number(await this.library.getBalance(this.account))
+  }
```

Comment:
In the future this should be in Trucoins, right?

Reply:
Yep!, maybe `getBalance` use format from ERC20 contract. we have to do the test.

I going to delete EtherJS class, at this moment we are not using it because we use `wagmi lib`